index.html

```diff
@@ -267,12 +267,20 @@
   <body>
     <section id='abstract'>
       <p>
-[=Credentials=] are integral to our daily lives; driver's licenses confirm
-our capability to operate motor vehicles, university degrees assert our level
-of education, and government-issued passports permit travel between countries.
-This specification provides a mechanism to express these sorts of
-[=credentials=] on the Web in cryptographically secure, privacy-respecting,
-and machine-verifiable way.
+This specification provides normative requirements and non-normative guidance
+for implementing and managing [=verifiable credentials=] and associated
```

Comment:
I prefer `expressing` over `implementing and managing`. I think it aligns more with the fact that this spec is a for a data model.

Reply:
@wip-abramson — Maybe this?
```suggestion
for expressing and consuming [=verifiable credentials=] and associated
```